lib/stm32/h7/Makefile

```diff
@@ -42,7 +42,7 @@ OBJS += flash_common_all.o flash_common_f.o flash_common_f24.o
 OBJS += fmc_common_f47.o
 OBJS += gpio_common_all.o gpio_common_f0234.o
 OBJS += pwr.o rcc.o
-OBJS += rcc_common_all.o
+OBJS += rcc_common_all.o rng_common_v1.o
```

Comment:
separate lines please, exactly for this reason, it reduces merge conflicts.  only things on same lines are same peripheral

Reply:
Would you like me to split the other lines while we're at it? (Or is it that things related to the same peripheral can be kept together?)